### PR TITLE
ILS fix-06/03/18

### PR DESCRIPTION
**Pull request by orangejuicie**

### CODE REVIEW

File: Glideslope.json [hunk 1]
```diff
@@ -19126,41 +19126,41 @@
...
+    "elevation": 0, 
     "name": "GS", 
-    "longitude": -90.254661, 
-    "associatedRunwayNumber": "28", 
+    "longitude": -90.25097379, 
+    "associatedRunwayNumber": "29", 
     "frequency": 109900, 
-    "glideslope": 3.0028556, 
-    "latitude": 29.991106, 
+    "glideslope": 3.002856,
```
Comment (CptNathanHope) [inline Glideslope.json]: Bearing does not match here

Comment (orangejuicie) [reply Glideslope.json]: Thanks.
I'll fix that.

File: LOC.json [hunk 2]
```diff
@@ -19202,43 +19202,56 @@
...
+    "airportICAO": "KMSY", 
+    "elevation": 0, 
+    "name": "LOC", 
+    "longitude": -90.24551065, 
+    "associatedRunwayNumber": "20", 
+    "frequency": 111700, 
+    "latitude": 30.00305238, 
+    "identifier": "IONW", 
+    "type": 4
+  },
```
Comment (CptNathanHope) [inline LOC.json]: LOC approaches should be added to the bottom of the file with the others. Keep in alphabetical order.

File: LOC.json [hunk 3]
```diff
@@ -19202,43 +19202,56 @@
...
+    "bearing": 195.41, 
+    "airportICAO": "KMSY", 
+    "elevation": 0, 
+    "name": "LOC", 
+    "longitude": -90.24551065, 
+    "associatedRunwayNumber": "20", 
+    "frequency": 111700, 
+    "latitude": 30.00305238, 
+    "identifier": "IONW", 
+    "type": 4
```
Comment (CptNathanHope) [inline LOC.json]: Change the type to 5.

**Comment by CptNathanHope** [comment]
Thanks for addressing the errors. Will add to repo, nice work. :)

**Pull request merged**